lambda/readPass/index.js

```diff
@@ -47,6 +47,19 @@ exports.handler = async (event, context) => {
         queryObj.ExpressionAttributeValues[':theDate'] = AWS.DynamoDB.Converter.input(dateselector);
         queryObj.FilterExpression += ' AND contains(#theDate, :theDate)';
       }
+      // Filter Multiple Statuses
+      if (event.queryStringParameters.passStatus) {
+        const statusList = event.queryStringParameters.passStatus.split(',');
+        const statusObj = {};
+        statusList.forEach((status, index) => {
+          const statusName = ":passStatus" + index;
+          statusObj[statusName.toString()] = AWS.DynamoDB.Converter.input(status);
+        });
```

Comment:
Prefer for/of instead of forEach as forEach is not async/await compatible (not that you're doing any of that here) also it's much more performant :)

Reply:
refactored, addressed in `3f1e98a`